### ISSUE TITLE
New submissions for Thu,  6 Feb 20

### ISSUE THREAD

**Issue by kobiso** [kobiso/daily-arxiv-noti#15]
### Context-Aware Recommendations for Televisions Using Deep Embeddings with  Relaxed N-Pairs Loss Objective
 - **Authors:** Miklas S. Kristoffersen, Sven E. Shepstone, Zheng-Hua Tan
 - **Subjects:** Information Retrieval (cs.IR)
 - **Arxiv link:** https://arxiv.org/abs/2002.01554
 - **Pdf link:** https://arxiv.org/pdf/2002.01554
 - **Abstract**
 This paper studies context-aware recommendations in the television domain by proposing a deep learning-based method for learning joint context-content embeddings (JCCE). The method builds on recent developments within recommendations using latent representations and deep metric learning, in order to effectively represent contextual settings of viewing situations as well as available content in a shared latent space. This embedding space is used for exploring relevant content in various viewing settings by applying an N -pairs loss objective as well as a relaxed variant introduced in this paper. Experiments on two datasets confirm the recommendation ability of JCCE, achieving improvements when compared to state-of-the-art methods. Further experiments display useful structures in the learned embeddings that can be used to gain valuable knowledge of underlying variables in the relationship between contextual settings and content properties.
### Enhancing Feature Invariance with Learned Image Transformations for  Image Retrieval
 - **Authors:** Osman Tursun, Simon Denman, Sridha Sridharan, Clinton Fookes
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2002.01642
 - **Pdf link:** https://arxiv.org/pdf/2002.01642
 - **Abstract**
 Off-the-shelf convolutional neural network features achieve state-of-the-art results in many image retrieval tasks. However, their invariance is pre-defined by the network architecture and training data. In this work, we propose using features aggregated from transformed images to increase the invariance of off-the-shelf features without fine-tuning or modifying the network. We learn an ensemble of beneficial image transformations through reinforcement learning in an efficient way. Experiment results show the learned ensemble of transformations is effective and transferable.
## Keyword: face recognition
There is no result 
### Large Batch Training Does Not Need Warmup
 - **Authors:** Zhouyuan Huo, Bin Gu, Heng Huang
 - **Subjects:** Machine Learning (cs.LG); Machine Learning (stat.ML)
 - **Arxiv link:** https://arxiv.org/abs/2002.01576
 - **Pdf link:** https://arxiv.org/pdf/2002.01576
 - **Abstract**
 Training deep neural networks using a large batch size has shown promising results and benefits many real-world applications. However, the optimizer converges slowly at early epochs and there is a gap between large-batch deep learning optimization heuristics and theoretical underpinnings. In this paper, we propose a novel Complete Layer-wise Adaptive Rate Scaling (CLARS) algorithm for large-batch training. We also analyze the convergence rate of the proposed method by introducing a new fine-grained analysis of gradient-based methods. Based on our analysis, we bridge the gap and illustrate the theoretical insights for three popular large-batch training techniques, including linear learning rate scaling, gradual warmup, and layer-wise adaptive rate scaling. Extensive experiments demonstrate that the proposed algorithm outperforms gradual warmup technique by a large margin and defeats the convergence of the state-of-the-art large-batch optimizer in training advanced deep neural networks (ResNet, DenseNet, MobileNet) on ImageNet dataset.
### Analyzing the Dependency of ConvNets on Spatial Information
 - **Authors:** Yue Fan, Yongqin Xian, Max Maria Losch, Bernt Schiele
 - **Subjects:** Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2002.01827
 - **Pdf link:** https://arxiv.org/pdf/2002.01827
 - **Abstract**
 Intuitively, image classification should profit from using spatial information. Recent work, however, suggests that this might be overrated in standard CNNs. In this paper, we are pushing the envelope and aim to further investigate the reliance on spatial information. We propose spatial shuffling and GAP+FC to destroy spatial information during both training and testing phases. Interestingly, we observe that spatial information can be deleted from later layers with small performance drops, which indicates spatial information at later layers is not necessary for good performance. For example, test accuracy of VGG-16 only drops by 0.03% and 2.66% with spatial information completely removed from the last 30% and 53% layers on CIFAR100, respectively. Evaluation on several object recognition datasets (CIFAR100, Small-ImageNet, ImageNet) with a wide range of CNN architectures (VGG16, ResNet50, ResNet152) shows an overall consistent pattern.
## Keyword: cars196
There is no result
